scripts/performance/perf_load/perf_client.py

```diff
@@ -236,8 +325,16 @@ def read_cb(self):
             self._stat.reply(req_data, e)
             self._loop.stop()
             raise e
+
         await self._req_generator.on_request_generated(req_data, sig_req)
 
+    async def append_taa_acceptance(self, req):
+        if self._taa_text == "":
+            return req
+
+        return await ledger.append_txn_author_agreement_acceptance_to_request(
```

Comment:
If this is XFER request, then TAA metadata is added a bit differently.
```
extra = prepare_payment_extra_with_acceptance_data(extra, text, version, None, mechanism, round(time.time()))
indy_build_payment_req = (...., extra=extra)
```

Reply:
What's worse - in case of XFER requests this needs to be done as part of transaction creation, not afterwards. For now I created a dirty fix, will improve in next PR.